src/control/cmd/dmg/pool_test.go

```diff
@@ -251,6 +251,45 @@ func TestPoolCommands(t *testing.T) {
 			}, " "),
 			nil,
 		},
+		/* TODO: Tests need to be fixed after pull pool info */
+		{
+			"Extend pool with missing arguments",
+			"pool extend",
+			"",
+			errMissingFlag,
+		},
+		{
+			"Extend a pool with a single rank",
+			fmt.Sprintf("pool extend --pool 031bcaf8-f0f5-42ef-b3c5-ee048676dceb --ranks=1 --scm-size %s", testScmSizeStr),
+			strings.Join([]string{
+				"ConnectClients",
+				printRequest(t, &control.PoolExtendReq{
+					UUID:      "031bcaf8-f0f5-42ef-b3c5-ee048676dceb",
+					Ranks:     []uint32{1},
+					ScmBytes:  uint64(testScmSize),
+					User:      eUsr.Username + "@",
+					UserGroup: eGrp.Name + "@",
+					Sys:       "daos_server",
+				}),
+			}, " "),
+			nil,
+		},
+		{
+			"Extend a pool with multiple ranks",
+			fmt.Sprintf("pool extend --pool 031bcaf8-f0f5-42ef-b3c5-ee048676dceb --ranks=1,2,3 --scm-size %s", testScmSizeStr),
+			strings.Join([]string{
+				"ConnectClients",
+				printRequest(t, &control.PoolExtendReq{
+					UUID:      "031bcaf8-f0f5-42ef-b3c5-ee048676dceb",
+					Ranks:     []uint32{1, 2, 3},
```

Comment:
this will be `system.Ranks` as opposed to `uint32` as per the change mentioned in previous comment